lib/hammer_cli_csv/content_views.rb

```diff
@@ -9,6 +9,9 @@ class ContentViewsCommand < BaseCommand
       command_name 'content-views'
       desc         'import or export content-views'
 
+      option %w(--no-publish), :flag, _('Skip publishing content views on import')
+      option %w(--no-promote), :flag, _('Skip promoting content views on import')
```

Comment:
Ah, I overlooked your last sentence. I think I will go with updating the descriptions.